warehouse/internal/repo/staging.go

```diff
@@ -293,3 +302,70 @@ func (repo *StagingFiles) GetAfterID(ctx context.Context, sourceID, destinationI
 
 	return repo.parseRows(rows)
 }
+
+func (repo *StagingFiles) SetStatuses(ctx context.Context, ids []int64, status string) (err error) {
```

Comment:
```suggestion
func (repo *StagingFiles) SetStatuses(ctx context.Context, ids []int64, status string) error) {
```
Since we are wrapping `error` at every return, Should we avoid variable shadowing?